### ISSUE TITLE
[Question] Chaturabte "Online Favorites"

### ISSUE THREAD

**Issue by Kodionpi** [dobbelina/repository.dobbelina#606]
Am I missing something here? What does "Online Favorites" do in Chaturbate? When I click on it, it loads for a very long time, but then only a blank page appears.

Does this mean the "followed cams"? -but there would have to be a log in to Chaturbate within Cummination for that.

**Comment by TarkusLV** [comment]
It's those Chaturbate models you have in your Favorites that are currently online.

**Comment by Kodionpi** [comment]
I see, thank you. Is it possible that we get a direct login option,- so that one can log into the Chaturbate account and access all the saved "followed cams"?

I understand now that I can add any Models using the "Online Favorite" feature, but I'm more interested in being able to access all the Models already connected to the Chaturbate account.

**Comment by TarkusLV** [comment]
It's not possible now (in that it's not coded), but I'm not a developer, so I don't know if that's possible for the future.

**Comment by Kodionpi** [comment]
Okay. 

Regarding the current options. When I add a stream to favourites, it appears in the Cummination "Favourite Videos" on the frontpage. -but not under "Online Favourites" within Chaturbate. What am I doing wrong? Possibly because I add the stream but not the Model? Is there a difference?

**Comment by TarkusLV** [comment]
Have you restarted Kodi?  It's not very dynamic, and is mainly only useful when first starting Kodi.  Or at least when opening Cumination the first time.  After that it doesn't seem to get updated (at least not in real time).

**Comment by Kodionpi** [comment]
Yes, I rebooted. -but unfortunately it doesn't change anything-when I click on "Online Favorites" within Chaturbate, it loads very long and ends with an error and I'm thrown back to the previous page.
